app/react/src/server/cra-config.js

```diff
@@ -81,8 +81,7 @@ export const getTypeScriptRules = (webpackConfigRules, configDir) => {
 export function getCraWebpackConfig(mode) {
   const pathToReactScripts = getReactScriptsPath();
 
-  const craWebpackConfig =
-    mode === 'production' ? 'config/webpack.config.prod' : 'config/webpack.config.dev';
+  const craWebpackConfig = 'config/webpack.config';
```

Comment:
I guess I shouldn't do that due to backward compatibility with older versions of CRA and react-scripts

Reply:
I've added a check for this. Let's see if CI passes.